website_enterprise_remove/views/res_config_view.xml

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<!--
+    Copyright 2016 LasLabs Inc.
+    License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+-->
+
+<odoo>
+
+    <record id="view_website_config_settings" model="ir.ui.view">
+        <field name="name">Website settings</field>
+        <field name="model">website.config.settings</field>
+        <field name="inherit_id" ref="website.view_website_config_settings"/>
+        <field name="arch" type="xml">
+            <xpath expr="//div[field[@name='module_website_form_editor']]/../preceding-sibling::*[1]"
+                position="attributes">
+                <attribute name="invisible">True</attribute>
+            </xpath>
+            <xpath expr="//div[field[@name='module_website_form_editor']]/.." position="attributes">
```

Comment:
As per the config structure:
[res_config](https://github.com/odoo/odoo/blob/9.0/addons/website/views/res_config.xml#L91)

Instead of targeting each separate div containing each field with its corresponding label, I opted to target the parent div if all the settings were marked as enterprise in that group. The module_website_form_editor is the first div group item in the group, so I just used that to target the parent div (the div that's sibling to label string="Extra Features"), which if set to invisible, covers the a/b test setting as well.